e2etest/zt_newe2e_list_test.go

```diff
@@ -90,6 +130,51 @@ func (s *ListSuite) Scenario_ListRunningTally(svm *ScenarioVariationManager) {
 	ValidateListOutput(svm, stdout, expectedObjects, expectedSummary)
 }
 
+func (s *ListSuite) Scenario_ListVersionIdNoAdditionalVersionsText(svm *ScenarioVariationManager) {
```

Comment:
If we are testing the text output for all the properties, we should test more properties than just 1 I think.

Reply:
Ditto